charts/mirrorbits-parent/values.yaml

```diff
@@ -78,3 +78,7 @@ httpd:
   backendServiceNameTpl: '{{ include "httpd.fullname" (index $.Subcharts "httpd")}}'
 rsyncd:
   enabled: false
+serviceaccount:
+  enabled: false
+  ## uncomment to reuse an existing service account
+  # existingServiceAccount: jenkins-infra:mirrorbits
```

Comment:
The `existingServiceAccount` was a use case for Jenkins agents only , when the Jenkins helm-chart was handling the creation of a  SVCaccount **before** our custom chart was installed.

Unless e have a use case here, I propose to delete every reference to existingServiceAccount